modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -87,11 +91,12 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 //====
 //<6> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 //endif::openshift-origin[]
-<4> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<5> The name of an existing resource group.
-<6> Required. The installation program prompts you for this value.
+<4> The smtLevel specifies the level of SMT to set to the control plane and compute machines. The supported values are 1, 2, 3, 4, 5, 6, 7, 8, 'off' and 'on'. The smtLevel 'off' sets SMT to off and smtlevel 'on' sets SMT to 8 on the cluster nodes. For smtLevel 1 and 2, more number of processor cores need to be assigned to the cluster nodes while deploying. Ensure that the number of 'processors' set in the `install-config.yaml` file meets the requirements for deploying {product-title} successfully. One vCPU is equivalent to one physical core when simultaneous multithreading (SMT), or hyperthreading, is not enabled. When enabled, use the following formula to calculate the corresponding ratio: (threads per core × cores) × sockets = vCPUs.
```

Comment:
We want to be more generic than just calling out smtLevel 1 and 2.  We should make a generic comment that when you use a smaller smtLevel, you may need to assign more cores.  Total vCPUs is computed as threads per core*number of cores*number of sockets.

Reply:
This formula `(threads per core × cores) × sockets = vCPUs.` looks too generic, IMO we should use the powervm terminologies(need to dig deeper where to get that information)

Reply:
also need to include what's the default value, add some examples what happens user doesn't specific anything and for the specific SMT value.

Reply:
> This formula (threads per core × cores) × sockets = vCPUs. looks too generic, IMO we should use the powervm terminologies(need to dig deeper where to get that information)

@mkumatag 
Referring to the output of `lscpu` command on the cluster nodes for terminology to be used in the documentation ([from QE task](https://github.ibm.com/redstack-power/project-mgmt/issues/3764#issuecomment-72125540)):
```
sh-5.1# lscpu
Architecture:            ppc64le
  Byte Order:            Little Endian
CPU(s):                  8
  On-line CPU(s) list:   0-7
Model name:              POWER9 (architected), altivec supported
  Model:                 1.3 (pvr 004e 2103)
  Thread(s) per core:    8
  Core(s) per socket:    1
  Socket(s):             1
  Physical sockets:      16
  Physical chips:        1
  Physical cores/chip:   10
Virtualization features:
  Hypervisor vendor:     pHyp
  Virtualization type:   para
Caches (sum of all):
  L1d:                   64 KiB (2 instances)
  L1i:                   64 KiB (2 instances)
  L2:                    512 KiB (1 instance)
  L3:                    10 MiB (1 instance)
NUMA:
  NUMA node(s):          1
  NUMA node2 CPU(s):     0-7
```

Reply:
interesting!